docs/journalist/introduction.rst

```diff
@@ -6,5 +6,84 @@ Introduction
 What is Qubes OS?
 -----------------
 
+`Qubes OS`_ is an open source, security-focused
+operating system. It is very different than operating systems you may be
+familiar with already, because the operating system consists of multiple
+isolated virtual machines (called qubes), which allow you to separate more
+trusted components, files, or programs on your computer from less trusted
+components, files, or programs.
+
+Broadly speaking, this means that even if one of your qubes is
+exposed to malware, your other qubes can still be protected, which is not true
+of other operating systems.
+
+.. _`Qubes OS`: https://www.qubes-os.org
+
+How is using Qubes different from using virtual machines?
```

Comment:
I really struggled over both those things. 
1. Do you think type 1/type 2 belongs in the journo guide? I felt like it was a bit of a rabbithole for folks wanting to understand at the basic level how to use the system. I would say this belongs in the security FAQ.
2. qube vs VM: I don't like "qube" but was trying to introduce the Qubes terminology gently, in case it is encountered anywhere else (in the Qubes documentation, for example). I feel less strongly about this and can stick to VM if desired.

Reply:
I get your point on no.1 . For  no. 2, my vote would be to standardize on VM.

Reply:
fixed in 1ef26a8 - references to "qubes" (lc) are now "VM" or "virtual machine"

Reply:
IMO this section should be moved to the FAQ, and cross-referenced from the intro. It is too technical for the introductory chapter.

Reply:
moved and cross-referenced in 5eda33f!

Reply:
:heavy_check_mark: